### PR TITLE
drivers: added driver for PCD8544 LCD display

### PR DESCRIPTION
**Pull request by haukepetersen**
introducing simple display support to RIOT :-)

The PCD8544 displays are the ones that were once used in the old Nokia 5510 cell phones and now they are available for little money (around ~1-2$) at the usual sources. Have fun!

### CODE REVIEW

File: drivers/include/pcd8544.h [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+ *
+ * @param[in] dev           device descriptor of display to use
+ * @param[in] x             column to put the character [0 - 13]
+ * @param[in] y             row to put the character [0 - 5]
+ * @param[in] c             ASCII code for the character to write
+ */
+void pcd8544_write_c(pcd8544_t *dev, uint8_t x, uint8_t y, const char c);
+
+/**
+ * @brief   Print the RIOT logo on the screen
```
Comment (OlegHahm) [inline drivers/include/pcd8544.h]: This `brief` doesn't match the remaining description.

File: drivers/pcd8544/Makefile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+MODULE = pcd8544
```
Comment (OlegHahm) [inline drivers/pcd8544/Makefile]: superfluous

File: drivers/pcd8544/pcd8544.c [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0xfc, 0xff, 0xff, 0xff, 0xff,
+    0x07, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0xff, 0xff,
+    0xff, 0xff, 0xff, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```
Comment (OlegHahm) [inline drivers/pcd8544/pcd8544.c]: I think this is wrong. Please fix!!!elf

Comment (haukepetersen) [inline drivers/pcd8544/pcd8544.c]: Are you a golfer?

**Comment by haukepetersen** [comment]
fixed (reasonable) comments

**Comment by OlegHahm** [comment]
ACK

**Comment by haukepetersen** [comment]
removed one newline from the end of `pcd8544.c`

**Comment by OlegHahm** [comment]
`file has an unknown license header: 'drivers/pcd8544/pcd8544.c'`

**Comment by haukepetersen** [comment]
ups, that was a very old header... fixed.

**Pull request merged**